torchbenchmark/util/backends/torchdynamo.py

```diff
@@ -0,0 +1,49 @@
+"""
+Support TorchDynamo(https://github.com/facebookresearch/torchdynamo) backends
+"""
+import argparse
+from typing import List
+
+class NullContext:
+    def __enter__(self):
+        pass
+
+    def __exit__(self, exc_type, exc_val, exc_tb):
+        pass
+
+def help(fn):
+    return fn.__doc__
+
+def null_experiment(args, model_iter_fn, model, example_inputs):
+    """
+    A no-op experiment useful for making sure TorchBenchmark alone works properly.
+    """
+
+    return []
+
+def parse_torchdynamo_args(model: 'torchbenchmark.util.model.BenchmarkModel', dyamo_args: List[str]) -> argparse.Namespace:
+    parser = argparse.ArgumentParser()
+    group = parser.add_mutually_exclusive_group()
+    group.add_argument("--nothing", action="store_true", help=help(null_experiment))
+    group.add_argument(
+        "--nops",
+        action="store_true",
+        help="Test that bytecode rewriting works properly.",
+    )
+    parser.add_argument(
+        "--nopython", action="store_true", help="Turn graph breaks into errors"
+    )
+    args = parser.parse_args(dyamo_args)
+    return args
+
+def apply_torchdynamo_args(model: 'torchbenchmark.util.model.BenchmarkModel', args: argparse.Namespace):
+    import torchdynamo
+    optimize_ctx = NullContext()
+    if args.nothing:
+        pass
+    elif args.nops:
+        optimize_ctx = torchdynamo.eval_frame._optimize_catch_errors(
+            torchdynamo.testing.debug_insert_nops, nopython=args.nopython
+        )
+    model.add_context(lambda: optimize_ctx)
```

Comment:
I am curious if you just replace this with:
`lambda: torchdynamo.optimize("ofi")`
do you see a speedup on any model for inference?

I'd expect that one line is all you need.

There are currently 27 choices you could replace "ofi" with, see: https://github.com/facebookresearch/torchdynamo/issues/58

Reply:
@jansel I see a speedup when using ofi as the backend, but I also see a strange error `jit error` because `example_inputs` is `None`, which it shouldn't be.

Reply:
Hrm, I'll look into that error.

Reply:
@xuzhao9 can you retry with the latest TorchDynamo.  This patch should fix it: https://github.com/facebookresearch/torchdynamo/commit/86a2bea62cebedb51d5a1f2230e7817ff2f786fb

Reply:
@jansel Thanks! I confirm the latest torchdynamo fixes this issue.